### PR TITLE
Testbranch jay

### PR DESCRIPTION
**Pull request by JayMan91**
I created FirstUsage.md, as a guide to use python package, documentation and testing for first time users like me.

### CODE REVIEW

File: FirstUsage.md [hunk 1]
```diff
@@ -36,11 +36,11 @@ setup(
     author='Tias Guns',
     author_email='',
     license='MIT',
-    # packages=find_packages(exclude=['sample_app.test']),
+    # packages=find_packages(exclude=['mccpy.test']),
```
Comment (tias) [inline FirstUsage.md:39]: should be new name: cpmpy

File: FirstUsage.md [hunk 2]
```diff
@@ -24,18 +24,70 @@ To use markdown the documentation says we have to `pip install recommonmark` and
...
 extensions = ['recommonmark']
 ```
-# Testing using tox https://tox.readthedocs.io/en/latest/
+# Building package Locally
+First create a `setup.py` file
+```python
+from setuptools import setup, find_packages
+
+setup(
+    name='mcppy',
```
Comment (tias) [inline FirstUsage.md:33]: new name

File: FirstUsage.md [hunk 3]
```diff
@@ -24,18 +24,70 @@ To use markdown the documentation says we have to `pip install recommonmark` and
...
+numpy>=1.20.0
+```
+Create a folder named `CPModules` (or whatever) and write all the class and definition inside it within `model.py` (or whatever).
+Now to install in test mode, first create a virtual environment. and then run after going to directory containing the `setup.py`
+```
+pip install -e .
+```
+To check whether the installation is successful, open python
+```python
+form CPModules.model import *
```
Comment (tias) [inline FirstUsage.md:59]: should be 'from cpmpy import *'?

**Review by tias — CHANGES_REQUESTED** [review]
Can you rebase on current master with new name?

File: FirstUsage.md (hunk 3 above)
Comment (JayMan91) [reply FirstUsage.md]: It's the name of the folder which in this case `cpmodules`, I think a good practice is to give it a same name as the package, but I d don't know

**Pull request merged**